editor/editor_inspector.cpp

```diff
@@ -4664,6 +4664,7 @@ EditorInspector::EditorInspector() {
 	search_box = nullptr;
 	_prop_edited = "property_edited";
 	set_process(false);
+	set_focus_mode(FocusMode::FOCUS_ALL);
```

Comment:
I meant this line is unnecessary. The inspector will display border when any child is focused.

Reply:
I tested it removing that line and the inspector didn't get the focus border drawn anymore

Reply:
I tested and it draws focus when its child is focused
![image](https://github.com/user-attachments/assets/7a958799-8a96-46a2-9f1f-669e2fdf0a87)
which is what I'd expect.

Reply:
Now I'm getting the problem: in my tests I always clicked on the labels (like Background Offset). For me, clicking on any place inside of the Inspector should draw the focus border. But to be honest I don't have an strong opinion on that, so I'll remove it.
Thanks for the explanation and the screenshot! (it was really helpful to understand what was happening)

Reply:
Yeah the property labels aren't actually getting focused.

Reply:
Ok sorry I totally missed that clicking labels would focus the inspector.
In that case it was fine.